### PR TITLE
Collection of typos

### PR DESCRIPTION
**Pull request by korverdev**
This is a collection of typos identified in code reviews by my team over the past few months. I've also attempted to add variants of these words to the best of my ability. Huge thank you to the maintainers of this project for solving a very real need for us! We're at the point where roughly 90% of typo-related errors are being caught automatically.

Additionally, I've updated the `allk` misspelling to possibly also correct to `ally`.

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -13223,6 +13224,8 @@ conervative->conservative
 conesencus->consensus
 conet->connect
 coneted->connected
+conetent->content
+conetents->contents
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:13228]: Perhaps add
```
conetented->contented
conetenting->contenting
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -19612,6 +19615,8 @@ disabing->disabling
 disabl->disable
 disablle->disable
 disadvantadge->disadvantage
+disaggretate->disaggregate
+disaggretated->disaggregated
```
Comment (DimitriPapadopoulos) [inline codespell_lib/data/dictionary.txt:19619]: Perhaps add
```
disaggretates->disaggregates
disaggretating->disaggregating
```

**Comment by korverdev** [comment]
@DimitriPapadopoulos Thanks for weighing in on these changes! This should be ready for another look whenever you have a few minutes to spare.

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (korverdev) [reply codespell_lib/data/dictionary.txt:13228]: Thanks for pointing that out! In this context, "content" meant material or substance. I didn't even think of the other meaning of the word.

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
@korverdev Thank you for your contributions.

**Pull request merged**